centreon/tests/e2e/features/Api-Token/01-api-token-create.feature

```diff
@@ -0,0 +1,39 @@
+@ignore
+@REQ_MON24865
+Feature: Create and Manage Basic API Token
+
+  As an administrator
+  I want to create a functional basic API token using the "Create new token" button
+  So that I can manage access to resources and services
+
+  Background:
+    Given I am logged in as an administrator
+    And I am on the API tokens page
+
+  Scenario: Create Basic API Token with Required Fields
+    When I click on the "Create new token" button
+    And I fill in the following required fields:
+      | Field     | Value                  |
+      | Name      | [Enter token name]     |
+      | User      | [Enter linked user]    |
+    And I select the duration as "30d"
+    And I click on the "Generate token" button
+    Then a new basic API token with hidden display should be generated
+
+  Scenario: Display and Copy Generated Token
+    Given a basic API token is generated
+    When I click to reveal the token
+    Then the token should be displayed and can be copied to the clipboard
```

Comment:
is the "copy to clipboard" action automatic when you reveal the token, or does it require additional human action (such as pushing more buttons?)